package.json

```diff
@@ -135,6 +134,14 @@
   },
   "volta": {
     "node": "20.14.0",
-    "pnpm": "9.2.0"
+    "pnpm": "9.14.2"
+  },
+  "pnpm": {
+    "overrides": {
+      "perf-primitives": "0.0.6"
+    },
```

Comment:
The error that I posted in https://github.com/yapplabs/ember-modal-dialog/pull/408#issuecomment-2504239833 occurred because [`liquid-wormhole@8cfb67b`](https://github.com/pzuraq/liquid-wormhole/commit/8cfb67b7fbf158f1e9187bbc2c47c10ee32b5a28) declared [a forked commit of `perf-primitives`](https://github.com/RobbieTheWagner/perf-primitives/commit/a6a26f11497ca27be3763a88a5f20744e424756b) as its dependency.

While it may be possible to configure this repo's GitHub Actions to allow installing dependencies from forks, for simplicity, I think it may be better to force the original `perf-primitives` to be installed.

It's worth noting that, because tests for the `animatable` route pass only with unofficial versions of `liquid-tether` and `liquid-wormhole`, a passing CI doesn't necessarily mean that this addon will work on a consuming project with `liquid-*` packages.